griptape/drivers/sql/amazon_redshift_sql_driver.py

```diff
@@ -101,4 +101,4 @@ def get_table_schema(self, table: str, schema: Optional[str] = None) -> Optional
         if self.database_credentials_secret_arn:
             function_kwargs["SecretArn"] = self.database_credentials_secret_arn
         response = self.client.describe_table(**function_kwargs)
-        return [col["name"] for col in response["ColumnList"]]
+        return str([col["name"] for col in response["ColumnList"]])
```

Comment:
This changes the functionality, previously it was a list, now it's a string.

Reply:
Should I change the return type of `get_table_schema()` then? Its currently returning `Optional[str]` which is the problem

Reply:
Oh great point, ok let's continue with this then.